src/SUMMARY.md

```diff
@@ -14,6 +14,13 @@
       - [Using the Data Catalog](cookbooks/analysis/data_catalog.md)
       - [Using the Glean Dictionary](cookbooks/analysis/glean_dictionary.md)
       - [Using the Probe Dictionary](cookbooks/analysis/probe_dictionary.md)
+    - [Data Quality](cookbooks/data_quality/intro.md)
+      - [Bigeye - Data Observability Platform](cookbooks/data_quality/index.md)
+      - [Intro to Bigeye](cookbooks/data_quality/intro.md)
```

Comment:
This is the same file as `Data Quality` above, making navigation weird.

Just remove this line. The intro page can be the overview page.